android/app/src/main/java/com/mattermost/share/RealPathUtil.java

```diff
@@ -45,9 +45,7 @@ public static String getRealPathFromURI(final Context context, final Uri uri) {
                         return id.replaceFirst("raw:", "");
                     }
                     try {
-                        final Uri contentUri = ContentUris.withAppendedId(
-                                Uri.parse("content://downloads/public_downloads"), Long.valueOf(id));
-                        return getDataColumn(context, contentUri, null, null);
+                        return getPathFromSavingTempFile(context, uri);
```

Comment:
What's the fix here? Were we just looking in the wrong location for these files?

Reply:
@hmhealey the fix here is that getDataColumn does not work for certain file types so now we create a temp file that can be read by our app.